commitlint.config.js

```diff
@@ -1 +0,0 @@
-module.exports = { extends: [ '@commitlint/config-conventional' ] };
```

Comment:
This file must exist and extend the new config file from `newspack-scripts`, otherwise test steps 9 (`commitizen`) and 10 (`commit-msg`) will fail.

```suggestion
module.exports = { extends: [ './node_modules/newspack-scripts/config/commitlint.config.js' ] };
```

Reply:
I can't reproduce this. With what error messages do steps 9 & 10 fail?

Reply:
```
git commit --allow-empty -am 'hehe'
git diff --staged
## Checking files, scope patches:
Skipping codeception since not configured
→ No staged files found.
⧗   input: hehe
✖   Please add rules to your `commitlint.config.js`
    - Getting started guide: https://git.io/fhHij
    - Example config: https://git.io/fhHip [empty-rules]

✖   found 1 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

The error comes from `commit-msg`, which is also run after commitizen. in step 9.

Reply:
Yeah, that's intended. `hehe` is a commit message that should fail, as described in step 10. `git commit --allow-empty -m 'feat: something'` should pass.

Reply:
The failure message isn't that the message doesn't satisfy the rules, though. It's that `commitlint` is missing rules entirely—a proper commit message also fails with the same message: `✖   Please add rules to your `commitlint.config.js``

Reply:
Ah you're right – I read your comment to quickly. I still can't reproduce that, though. I've also SSH'd into the CI machine to verify in more pure conditions, and was also unable to reproduce:

<img width="752" alt="image" src="https://user-images.githubusercontent.com/7383192/146795603-6e2f8060-d48b-4f67-86de-2eae939abd61.png">

_(Note that I'm not committing here, but using `commitlint` directly – this is because CI does not have git user configured)_

Reply:
After removing `node_modules` and `.cache`, I'm still getting the config error message when using `git commit`:

```zsh
➜  newspack-plugin git:(feat/use-newspack-scripts) git commit --allow-empty -m 'haha'
git diff --staged
## Checking files, scope patches:
Skipping codeception since not configured
→ No staged files found.
⧗   input: haha
✖   Please add rules to your `commitlint.config.js`
    - Getting started guide: https://git.io/fhHij
    - Example config: https://git.io/fhHip [empty-rules]

✖   found 1 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

However, running the `commitlint` script directly as in your last message does work:

```zsh
➜  newspack-plugin git:(feat/use-newspack-scripts) echo 'haha' | ./node_modules/.bin/newspack-scripts commitli
nt
⧗   input: hehe
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

Not sure what else I could try?

Reply:
Again, retaining `commitlint.config.js` and simply extending the `newspack-scripts` config [does work for me](https://github.com/Automattic/newspack-plugin/pull/1368#discussion_r769888323)—any objections to keeping it around?

Reply:
That's weird, in theory `git commit --allow-empty -m 'haha'` & `echo 'haha' | ./node_modules/.bin/newspack-scripts commitlint` should have the same effect (can't reproduce locally). I've added the config file back in e97da649dda357da31342ceaec1144e67aaa9f2e.

Reply:
Working for me now, thanks for reinstating!